components/avatar/avatar.css

```diff
@@ -0,0 +1,53 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap");
+body {
+  font-family: "Source Sans Pro", sans-serif;
+  font-family: "Ubuntu", sans-serif;
+}
+
+* {
+  --primary-color: rgb(14, 139, 45);
+  --secondry-color: rgb(204, 23, 23);
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.avatar {
+  width: 3rem;
+  height: 3rem;
+  border-radius: 50%;
+  margin-bottom: 1rem;
+}
+
+.size-large {
```

Comment:
you should use avatar-large, avatar-small as the naming here because size-large can cause conflict afterwards and also avatar-square as well.

Reply:
Agree with farhan